src/script.rs

```diff
@@ -204,13 +182,25 @@ impl<'a> ScriptInvocation<'a> {
     where
         C: crate::aio::ConnectionLike,
     {
-        let hash: String = cmd("SCRIPT")
-            .arg("LOAD")
-            .arg(self.script.code.as_bytes())
-            .query_async(con).await?;
+        let hash: String = self.load_cmd().query_async(con).await?;
 
         debug_assert_eq!(hash, self.script.hash);
 
         Ok(hash)
     }
+
+    fn load_cmd(&self) -> Cmd {
+        let mut cmd = cmd("SCRIPT");
```

Comment:
Style nit: I think we could just chain this directly all the way through instead of introducing a variable binding?

Reply:
Fair enough.